I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 17, in script
    pause
  File "renpy/common/000statements.rpy", line 455, in execute_pause
    renpy.pause()
OSError: Couldn't find file 'audio/chapelcandy.ogg'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 17, in script
    pause
  File "/home/cpu/renpy8/renpy/ast.py", line 2232, in execute
    self.call("execute")
  File "/home/cpu/renpy8/renpy/ast.py", line 2220, in call
    return renpy.statements.call(method, parsed, *args, **kwargs)
  File "/home/cpu/renpy8/renpy/statements.py", line 281, in call
    return method(parsed, *args, **kwargs)
  File "renpy/common/000statements.rpy", line 455, in execute_pause
    renpy.pause()
  File "/home/cpu/renpy8/renpy/exports.py", line 1533, in pause
    rv = renpy.ui.interact(mouse='pause', type='pause', roll_forward=roll_forward, pause=delay)
  File "/home/cpu/renpy8/renpy/ui.py", line 299, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "/home/cpu/renpy8/renpy/display/core.py", line 3377, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, **kwargs) # type: ignore
  File "/home/cpu/renpy8/renpy/display/core.py", line 4097, in interact_core
    renpy.audio.audio.periodic()
  File "/home/cpu/renpy8/renpy/audio/audio.py", line 1166, in periodic
    raise_(exc[0], exc[1], exc[2])
  File "/home/tom/ab/renpy-build/tmp/install.linux-x86_64/lib/python3.9/site-packages/future/utils/__init__.py", line 441, in raise_
  File "/home/cpu/renpy8/renpy/audio/audio.py", line 1144, in periodic_thread_main
    periodic_pass()
  File "/home/cpu/renpy8/renpy/audio/audio.py", line 1086, in periodic_pass
    c.periodic()
  File "/home/cpu/renpy8/renpy/audio/audio.py", line 517, in periodic
    topf = load(filename)
  File "/home/cpu/renpy8/renpy/audio/audio.py", line 70, in load
    rv = renpy.loader.load(fn)
  File "/home/cpu/renpy8/renpy/loader.py", line 820, in load
    raise IOError("Couldn't find file '%s'." % name)
OSError: Couldn't find file 'audio/chapelcandy.ogg'.

Linux-5.19.9-arch1-1-x86_64-with-glibc2.36 x86_64
Ren'Py 8.0.1.22070801
ptagain 1.0
Tue Sep 20 00:32:36 2022
